elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -456,6 +481,9 @@ def grade(element_html, data):
             elif feedback_type == 'first-wrong':
                 if first_wrong == -1:
                     feedback = FIRST_WRONG_FEEDBACK['incomplete']
+                elif display_distractor_feedback and student_answer[first_wrong]['tag'] is None:
+                    text = student_answer[first_wrong]['inner_html']
+                    feedback += FIRST_WRONG_FEEDBACK['distractor-feedback'].format(str(first_wrong + 1), text, distractor_feedback[text] or '')
```

Comment:
Currently, the way you have your `display-distractor-feedback` option set up, it will affect what feedback is shown during intermediate attempts at a problem, AND the feedback that is shown to students after their last attempt at a problem (when they are shown the correct answer). 

I think that these things represent different instructional design decisions and so they should be controlled separately from one another.

Also, up to this point, the feedback that students are given on intermediate attempts has been purposely vague, not telling them all the information about why their solution is wrong. If we are going to give them more specific feedback in the case then they chose a distractor, do we also want to give them more specific feedback for the other cases? (out of order, or unfinished block-group)? In my mind, the option `feedback=first_wrong` should mean that the only thing the student is told is which block is the first one that is wrong, and if we want to give them more information than that, it should be a different option, `feedback=???` I don't know what to call it though.

Reply:
In terms of thing to call this feedback here's a few things that come to mind:
* `first_wrong_verbose`
* `first_wrong_w_distractor_feedback`
* `first_wrong_w_feedback`
* `first_wrong_show_feedback`
Any of these, or perhaps a fork of one of these, look okay?

Reply:
I ended up going with first-wrong-verbose for the time being.